react/typings/global.d.ts

```diff
@@ -168,6 +184,7 @@ declare global {
     messages: RenderRuntime['messages']
     mode: EditorMode
     viewport: Viewport
+    setBlockData: (blockData: EditorProviderState['blockData']) => void
```

Comment:
[2] Isn't this the same BlockData defined in this file?

Reply:
Let's use the first thread to discuss that 🙂